google/cloud/apigeeregistry/v1/admin_service.proto

```diff
@@ -47,6 +48,17 @@ service Admin {
     };
   }
 
+  // MigrateDatabase attempts to migrate the database to the current schema.
+  rpc MigrateDatabase(MigrateDatabaseRequest) returns (google.longrunning.Operation) {
+    option (google.api.http) = {
+      post: "/v1/migrate"
```

Comment:
Minor; Would `/v1/migrateDatabase` would be more detailed and consistent with the rpc name?

ps: I'm not sure which way is better: /migrate or :migrate (as custom method), but I do found examples in [codesearch using /v1/migrate](https://source.corp.google.com/search?q=file:.*.proto$%20%22%2Fv1%2Fmigrate%22&sq=package:piper%20file:%2F%2Fdepot%2Fgoogle3%20-file:google3%2Fexperimental)

Reply:
I'm ok with that - for consistency we could either use "migrate" for both or "[Mm]igrateDatabase" for both. Do you have a preference? (I think I slightly prefer the more explicit "migratedatabase" but it also seems that "database" is superfluous)

Reply:
I'm okay with both, but migrateDatabase is slightly clear from my perspective

Reply:
OK! I've changed that to "migrateDatabase"